src/lambda/config/parameterUtils.ts

```diff
@@ -0,0 +1,81 @@
+/*!
```

Comment:
Consider moving this file out from the `lambda` folder

Reply:
I think it does belong in `lambda` since it deals with lambda-specific functionality, but I moved it to `lambda/utilities`.

Let's come up with a well-defined set of folders and criteria for each folder, to avoid ambiguities like this in the future.

✔️